### DISCUSSION TITLE
Welcome to opentelemetry-demo-webstore Discussions!

### DISCUSSION THREAD

**Discussion by tigrannajaryan** [open-telemetry/opentelemetry-demo#9] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by cartersocha** [comment]
💯

**Reply by tonycurcuru7** [reply]
My name is Andrew and I'm a I.T. Specialist with a degree in observation, Through the smoke and the screams do you ever wonder what it all really means?

**Comment by JaredTan95** [comment]
As https://github.com/open-telemetry/opentelemetry-demo-webstore/issues/87 mentioned, 
Do we have any plan about springboot/springcloud with JSP/servelet/webmvc framework to cover more tracing span cases?

**Reply by cartersocha** [reply]
Hey Jared, this was the discussion thread I was talking about: https://github.com/open-telemetry/opentelemetry-demo-webstore/discussions/25

You can add an architecture comment or suggestion on this doc: https://docs.google.com/document/d/1toQqAa4GApjFN3bvkEsq98Rb7WtbeKr3whsipAB3j1Q/edit

@austinlparker as an FYI

**Reply by JaredTan95** [reply]
> Hey Jared, this was the discussion thread I was talking about: #25
> 
> You can add an architecture comment or suggestion on this doc: https://docs.google.com/document/d/1toQqAa4GApjFN3bvkEsq98Rb7WtbeKr3whsipAB3j1Q/edit
> 
> @austinlparker as an FYI

got it.
